packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -93,7 +93,12 @@ export const bookingsRouter = router({
   get: authedProcedure
     .input(
       z.object({
-        status: z.enum(["upcoming", "recurring", "past", "cancelled", "unconfirmed"]),
+        filters: z.object({
+          teamIds: z.number().array().optional(),
+          userIds: z.number().array().optional(),
+          status: z.enum(["upcoming", "recurring", "past", "cancelled", "unconfirmed"]),
+          eventTypeIds: z.number().array().optional(),
```

Comment:
Need a new filter here as they are sent as arrays - the one from typequery isnt a true filter for the result and can create errors in the backend